workday/13.0/modules/ROOT/pages/workday-reference.adoc

```diff
@@ -0,0 +1,2689 @@
+= Workday Connector 13.0 Reference - Mule 4
+:page-aliases: connectors::workday/workday-reference.adoc
+
+
+Workday API version 35
+
+Anypoint Connector for Workday (Workday Connector) provides access to standard Workday operations.
+
+== Configurations
+---
+[[config]]
+=== Configuration
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<config_basic-authentication-api, Workday Basic Authentication Connection>> {nbsp}
```

Comment:
```suggestion
| Connection a| * <<config_basic-authentication-api, Workday Basic Authentication Connection>> 
```

Reply:
Did you run the clean up script to remove these fields {nbsp}?  No worries now, for the next time.